scripts/ops/SetupSolana/SetupSolana.s.sol

```diff
@@ -79,6 +79,7 @@ contract SetupSolana is DeployFraxOFTProtocol {
         broadcastConfig = nonEvmConfigs[0];
         broadcastConfigArray.push(nonEvmConfigs[0]);
 
+        // TODO : (Dhruvin) : update following addresses after solana deployment
```

Comment:
We will instead need to update peers here:
https://github.com/FraxFinance/frax-oft-upgradeable/blob/master/scripts/NonEvmPeers.json